core/controller/src/main/scala/whisk/core/entitlement/Entitlement.scala

```diff
@@ -82,8 +82,8 @@ protected[core] abstract class EntitlementProvider(config: WhiskConfig, loadBala
 
     private implicit val executionContext = actorSystem.dispatcher
 
-    private val invokeRateThrottler = new RateThrottler("actions per minute", config.actionInvokePerMinuteLimit.toInt)
-    private val triggerRateThrottler = new RateThrottler("triggers per minute", config.triggerFirePerMinuteLimit.toInt)
+    private val invokeRateThrottler = new RateThrottler("actions per minute", config.actionInvokePerMinuteLimit.toInt, _.limits.invocationsPerMinute)
```

Comment:
the _ is a nice convenience but it took me a pause to understand what's happening here (I had to check the signature for the rate throttler - not sure if it's worth being more verbose for clarity now... maybe.